project/SL510457/README.md

```diff
@@ -0,0 +1,82 @@
+# TourTracer : Christofides-TSP-Solver
+This proposal outlines the workflow and plans for the NYCU NSD term project in Autumn 2023.
+
+## Basic Information
+### Traveling Salesman Problem (TSP)
+TSP is a NP hard problem. Given a set of cities and the distance between every pair of cities, the problem is to find the shortest possible route that visits each city exactly once and returns back to the original city. 
```

Comment:
Algorithms in the algorithm textbooks are not usually good numerical problem.  If you want to find good problem for the project, it's better to consult textbooks of numerical analysis.

Reply:
Thank you for the feedback! I will look for other topics that are more suitable, and I will attend today's meetup for a discussion.